Sources/StreamChat/APIClient/APIClient_Tests.swift

```diff
@@ -78,7 +78,7 @@ class APIClient_Tests: StressTestCase {
         let testEndpoint = Endpoint<Data>.mock()
         
         // Create a request and assert the result is failure
-        let result = try await { apiClient.request(endpoint: testEndpoint, completion: $0) }
+        let result = try `await` { apiClient.request(endpoint: testEndpoint, completion: $0) }
```

Comment:
Shouldn't we rather rename it so something that won't cause the conflict before it will be possible to migrate? Like `_await` for example? :)

Reply:
Or even like `waitFor` so it's 100% unrelated to builtin `await` :D 
```swift
let result = try waitFor { apiClient.request(...) }
```

Reply:
![image](https://user-images.githubusercontent.com/6388510/117158261-8e23c600-adbf-11eb-8335-37375513d803.png)